config/config.go

```diff
@@ -226,9 +231,11 @@ func OptionExperimental(exp bool) Option {
 func OptionNetworkControlPlaneMTU(exp int) Option {
 	return func(c *Config) {
 		logrus.Debugf("Network Control Plane MTU: %d", exp)
-		if exp < 1500 {
-			// if exp == 0 the value won't be used
-			logrus.Warnf("Received a MTU of %d, this value is very low, the network control plane can misbehave", exp)
+		if exp < warningThNetworkControlPlaneMTU {
+			logrus.Warnf("Received a MTU of %d, this value is very low, the network control plane can misbehave (minimum 500)", exp)
```

Comment:
I would be more explicit and use 

`logrus.Warnf("Received a MTU of %d, this value is very low, the network control plane can misbehave, defaulting to minimum value (%d)", exp, minimumNetworkControlPlaneMTU)`